lib/ui/platform_dispatcher.dart

```diff
@@ -414,12 +414,10 @@ class PlatformDispatcher {
     }
   }
 
-  // If this value changes, update the encoding code in the following files:
-  //
-  //  * pointer_data.cc
-  //  * pointer.dart
-  //  * AndroidTouchProcessor.java
-  static const int _kPointerDataFieldCount = 35;
+  // This value must match kPointerDataFieldCount in pointer_data.cc. (The
+  // pointer_data.cc also listed out other locations that must be kept
+  // consistent.)
```

Comment:
Consider:

```suggestion
  // This value must match kPointerDataFieldCount in pointer_data.cc. (The
  // pointer_data.cc also lists other locations that must be kept consistent.)
```

I don't feel strongly, feel free to keep as is